api/src/mutations/drop.rs

```diff
@@ -655,6 +686,8 @@ pub struct CreateDropInput {
     pub blockchain: BlockchainEnum,
     pub creators: Vec<Creator>,
     pub metadata_json: MetadataJsonInput,
+    #[graphql(name = "type")]
+    pub drop_type: DropType,
```

Comment:
To make the API backward compatible can you make this optional and default to edition?